src/YourEasyRent/Services/State/UserStateManager.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace YourEasyRent.Services.State
+{
+    public class UserStateManager: IUserStateManager
+    {
+
+        public bool IsCategoryChosen { get; private set; }
+        public bool IsBrandChosen { get; set; }
```

Comment:
You might as well make all setters private

Reply:
I did it, but why should I do it?